smartcar/smartcar.py

```diff
@@ -141,27 +152,29 @@ def get_compatibility(
         client_secret = options.get("client_secret", client_secret)
         api_version = options.get("version", api_version)
 
-        if options.get("flags"):
-            flags_str = helpers.format_flag_query(options["flags"])
-            params["flags"] = flags_str
+        if api_version == "1.0":
+
+            if options.get("flags"):
+                flags_str = helpers.format_flag_query(options["flags"])
+                params["flags"] = flags_str
 
-        if options.get("version"):
-            api_version = options["version"]
+            if options.get("version"):
+                api_version = options["version"]
 
```

Comment:
This block is valid for all versions
- flags are supported on both versions
- you should be able to override your default API version no matter what you set it to

Reply:
I see, I'll fix this